### PR TITLE
Add: Error message when user attempts to create more than 10 targets

### PR DESCRIPTION
**Pull request by jpirazusta**
Screenshot:
Simulator Screen Shot - iPhone 11 - 2020-05-12 at 10 12 37

### CODE REVIEW

File: src/actions/targetActions.js [hunk 1]
```diff
@@ -24,4 +24,5 @@ export const deleteTarget = createThunk('DELETE_TARGET', targetService.deleteTar
 
 export const { success: getTargetsSuccess } = getTargets;
 export const { success: createTargetSuccess } = createTarget;
+export const { reset: createTargetReset } = createTarget;
```
Comment (ManuViola77) [inline src/actions/targetActions.js]: you can unify in just one sentence: 
`export const { success: createTargetSuccess,  reset: createTargetReset} = createTarget;`

**Review by ManuViola77 — APPROVED** [review]
👏

File: src/actions/targetActions.js (hunk 1 above)
Comment (jpirazusta) [reply src/actions/targetActions.js]: Oh, totally right. Thanks!

File: src/hooks/useCreateTargetForm.js [hunk 2]
```diff
@@ -2,11 +2,13 @@ import { useState, useEffect } from 'react';
...
 
 import { createTarget, getTopics } from 'actions/targetActions';
+import strings from 'locale';
 
 const useCreateTargetForm = (topic, emptyTopic, handleSubmit) => {
-  const { error: targetError, status: targetStatus } = useStatus(createTarget);
+  const { error, status: targetStatus } = useStatus(createTarget);
   const { error: getTopicsError, status: getTopicsStatus } = useStatus(getTopics);
   const [topicError, setTopicError] = useState();
+  const targetError = error ? strings.TARGET.maximumTargetsError : '';
```
Comment (elizabethlofredo) [inline src/hooks/useCreateTargetForm.js]: If there is another kind of error this will display always the maximum target message, right?

Comment (acollazomayer) [reply src/hooks/useCreateTargetForm.js]: I think the same

**Pull request merged**